### PR TITLE
[DA-2225] MIGRATION - Implement genomic_aw1_raw to genomic_set_member reconciliation.

### PR DESCRIPTION
**Pull request by j-kanuch**
## Description of changes/additions
This PR implements a new genomic cron job to reconcile the AW1 raw table (genomic_aw1_raw) to the `genomic_set_member` table. This PR:
1. Creates a weekly job to compare `genomic_aw1_raw` records to the records in `genomic_set_member`.
2. Writes data from `genomic_aw1_raw` to `genomic_set_member` for any missing data. Control samples are handled by creating a new genomic_set_member record.
3. Adds indexes to the `genomic_aw1_raw` table to increase performance on common joins—mainly for analysis.

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/dao/genomics_dao.py [hunk 1]
```diff
@@ -1966,6 +2026,27 @@ def get_raw_record_from_bid_genome_type(self, *, biobank_id, genome_type):
...
+            ).outerjoin(
+                GenomicSetMember,
+                GenomicSetMember.sampleId == GenomicAW1Raw.sample_id
+            ).filter(
+                GenomicSetMember.id.is_(None),
+                GenomicAW1Raw.ignore_flag == 0,
+                GenomicAW1Raw.biobank_id != "",
+                GenomicAW1Raw.sample_id != "",
+                GenomicAW1Raw.collection_tube_id != "",
+                GenomicAW1Raw.test_name != "",
```
Comment (dtharpeuno) [inline rdr_service/dao/genomics_dao.py:2042]: Should we lean into alchemy more and use .`isnot(None)` here?

File: rdr_service/offline/main.py [hunk 2]
```diff
@@ -902,6 +910,12 @@ def _build_pipeline_app():
         view_func=reconcile_gc_data_file_to_table,
         methods=["GET"]
     )
+    offline_app.add_url_rule(
+        OFFLINE_PREFIX + "ReconcileRawToAw1Ingested",
+        endpoint="reconcile_raw_to_aw1_ingested",
+        view_func=reconcile_raw_to_aw1_ingested,
+        methods=["GET"]
+    )
```
Comment (dtharpeuno) [inline rdr_service/offline/main.py:918]: Do you need to add this to the `cron` yaml files for the schedule?

Comment (j-kanuch) [reply rdr_service/offline/main.py:918]: yea, that's probably a good idea :)

File: rdr_service/dao/genomics_dao.py (hunk 1 above)
Comment (j-kanuch) [reply rdr_service/dao/genomics_dao.py:2042]: I'm specifically filtering for blank strings here.

Comment (dtharpeuno) [reply rdr_service/dao/genomics_dao.py:2042]: 👍

File: rdr_service/offline/main.py (hunk 2 above)
Comment (j-kanuch) [reply rdr_service/offline/main.py:918]: This has been added.

**Review by dtharpeuno — APPROVED** [review]

**Review by wangy70 — APPROVED** [review]

**Pull request merged**